swift-extensions/UIButtonExtensions.swift

```diff
@@ -20,7 +33,15 @@ extension UIButton {
                     button.updateBackgroundColor()
                 }))
 
-                addTarget(self, action: #selector(onButtonTouchUp), for: .touchUpInside)
+                observe(metaButton.onTap) { [weak self](tapAction: MetaAction) in
```

Comment:
Une raison de pas mettre cette observation dans `trikotInternalPublisherCancellableManager`?

Reply:
Ça le fait par défaut

Reply:
```
public func observe<V>(_ publisher: Publisher, toClosure closure: @escaping ((V) -> Void)) {
        observe(cancellableManager: trikotInternalPublisherCancellableManager, publisher: publisher, toClosure: closure)
    }
```

Reply:
Dans ce cas je m'explique mal la différence entre les assignations plus haut et celle-là, est-ce que celles du haut devraient être changés pour utiliser seulement `observe`? Il doit y avoir une raison au `KeyValueObservationHolder` je peux pas croire.

Reply:
The KeyValueObservationHolder is needed to invalidate an observe KeyPath

```
class KeyValueObservationHolder: NSObject, Cancellable {
    let kvo: NSKeyValueObservation

    init(_ kvo: NSKeyValueObservation) {
        self.kvo = kvo
    }

    public func cancel() {
        kvo.invalidate()
    }
}
```
Since in the case of the onTap we are observing a Publisher directly, it is not needed